uisdk/src/main/res/values/strings.xml

```diff
@@ -396,4 +396,5 @@
     <string name="pay_button_with_value">Pre-authorise %s</string>
     <!--  Loyalty  -->
     <string name="kh_uisdk_loyalty_balance_title">BALANCE: %s POINTS</string>
+    <string name="karhoo_uisdk_locale">en-GB</string>
```

Comment:
Same question as for the iOS version of the fix. What if it's en-US?

Reply:
The translations on the backend I think are in en-GB so we should be fine, if we support en-US we can add the resources for it and ensure the correct locale is passed